src/test/java/core/basesyntax/serviceimpl/ReaderServiceImplTest.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.serviceimpl;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import core.basesyntax.service.ReaderService;
+import java.util.List;
+import org.junit.Test;
+
+public class ReaderServiceImplTest {
+    private static final String INVALID_PATH =
+            "src/main/java/core/basesyntax/resources/12345.csv";
+    private static final String VALID_PATH =
+            "src/main/java/core/basesyntax/resources/transactions.csv";

```

Comment:
You should not use files from "src/main/java/core..." in your tests.
Create your test files: "src/test/resources/..."
Fix in other places